test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/OpenShiftLabel.java

```diff
@@ -299,7 +299,7 @@
 	public static class Others {
```

Comment:
Not related to this PR, but this seems like a strange use of a class.  Typical java convention is to stuff these into an interface, which means you can then add them to any class, remove the 'public' modifier